history/characters/320000_centaur.txt

```diff
@@ -2017,297 +2017,33 @@
 	572.1.14={ birth=yes }
 	640.8.9={ death=yes }
 }
-#Dynasty 126002
-320228={
```

Comment:
You deleted these characters, but they're still mentioned in `history` files:
![image](https://user-images.githubusercontent.com/46384992/57330049-077f7400-7126-11e9-9e6a-c0266ccf8050.png)